src/inverseRobot.js

```diff
@@ -24,7 +24,17 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const newRobot = {};
+
+  for (const [key, value] of Object.entries(robot)) {
+    if (!newRobot.hasOwnProperty(value)) {
+      newRobot[value] = key;
+    } else {
+      return null;
+    }
```

Comment:
invert this statements so you wouldn't use `else`
```suggestion
    if (newRobot.hasOwnProperty(value)) {
      return null;
    } 
    
    newRobot[value] = key;
```